data/game_teams_truncated_with_best_fans.csv

```diff
@@ -0,0 +1,13 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways

```

Comment:
good job on making different data files so we don't have to remake the test to fit the new information

Reply:
I second that